client/core/core.go

```diff
@@ -1422,6 +1423,9 @@ func (c *Core) initializeDEXConnections(crypter encrypt.Crypter) []*DEXBrief {
 				details := fmt.Sprintf("%s: %v", dc.acct.host, err)
 				c.notify(newFeePaymentNote("DEX auth error", details, db.ErrorLevel, dc.acct.host))
 				result.AuthErr = details
+				if strings.Contains(err.Error(), "rpc error: "+strconv.Itoa(msgjson.AccountNotFoundError)) {
+					result.AcctNotFound = true
+				}
```

Comment:
I did attempt to make this change without success. The errors.As(err, &mErr) always returns nil. It looks like a type incompatibility for some reason.

Reply:
I've left the attempt as commented code if you would like to dig deeper. Also the change did require adding 

```
func (e *Error) Error() string {
	return e.Message
}
```
to msgjson.types.go to satisfy the errors interface for errors.As()

Reply:
Did you change the places with `%w`? That is needed for wrapping the error. The errors change was in the diff. It would be better to return e.String() to get bot the code and the message in the error.

Reply:
I'm not sure what you're referring to on the %w. I rebased master. Maybe it came from there? I hope that the rebase is not ill-timed.